elt/synapse_gold.sql

```diff
@@ -153,3 +153,7 @@ CREATE VIEW IF NOT EXISTS synapse_data_warehouse.synapse.user_certified AS
   select ID, USER_NAME, FIRST_NAME, LAST_NAME, EMAIL, LOCATION, COMPANY, POSITION, PASSED
   from user_cert_joined
 ;
+
+// zero copy clone of processed access records
```

Comment:
Interesting that snowflake uses `//` for comments compared to the traditional `--` or `/* */` structure.

Reply:
Ya, it supports all of those, but Im going to standardize to `//`